cas/incident-report.md

```diff
@@ -1,169 +1,384 @@
+   - [Cause and Effect / Fishbone or Ishikawa](https://en.wikipedia.org/wiki/Ishikawa_diagram)
+   - [Drilling Down](https://sigma.software/about/media/problem-solving-techniques-part-two#2.-drill-down)
+   - SRE Handbook Guidance
+       - [Chapter 14](https://sre.google/sre-book/managing-incidents/)
+       - [Chapter 15](https://sre.google/sre-book/postmortem-culture/)
 
-YYYY-MM-DD:
-- HH:MM Example
+4. **Consider all potential contributing factors**. RCAs should consider a broad range of potential causes, including technical issues, policy issues, human factors, process breakdowns, and external influences. It's crucial to avoid jumping to conclusions or focusing solely on the most obvious cause(s).
 
-### Root Cause Analysis
+5. **Collect and analyze data**. Data is critical for supporting RCA conclusions. This might involve reviewing logs, monitoring metrics, internal incident reports, and other relevant information to identify patterns and anomalies.
 
+6. **Involve relevant stakeholders**. RCAs should be a collaborative effort involving engineers, operators, support teams, and other relevant stakeholders. This helps ensure a diverse range of perspectives and expertise is considered.
 
+7. **Prioritize action items**. The ultimate goal of an RCA is to prevent future incidents caused by the same failures. Therefore, it's important to identify and prioritize actionable recommendations that address the root causes identified.
 
-### Lessons Learned
+8. **Focus on blameless postmortems**. Emphasize a blameless culture when conducting postmortems. The focus should be on learning from the incident and improving systems, not on assigning blame or punishing individuals.
 
-#### What went well
+9. **Continuously improve the process**. RCA is an iterative process. Organizations should continuously refine their RCA approaches based on lessons learned from previous incidents and evolving best practices.
 
-* 
+### What is considered an ongoing commitment?
 
-#### What didn't go well
+An ongoing commitment is a formal pledge made by a CA Owner to address underlying issues and improve practices in response to an incident. These commitments extend beyond immediate incident remediation and demonstrate a sustained effort to enhance security, transparency, and/or accountability within the Web PKI ecosystem.
 
-* 
+Effective ongoing commitments consist of the following:
 
-#### Where we got lucky
+1. **Specificity**: The commitment clearly outlines what the CA Owner will do, including the intended outcome, target completion timeframe, and any relevant metrics or indicators.
+ 
+2. **Actionability**: The commitment describes concrete steps the CA Owner will take, ensuring it's not merely a statement of intent or a vague promise.
+ 
+3. **Measurability**: The commitment includes a way to track progress and assess whether the CA has fulfilled its pledge. This could involve public reporting, audits, or other verifiable means.
```

Comment:
I find this requirement quite confusing. How is "public reporting" (presumably by the CA) a form of measurability that is verifiable? Requiring external verifiability seems to greatly restrict actions the can be ongoing commitments. In the example below, I fail to see how "The presence of linting failures where certificates were issued more than 7 days after a new lint was released will represent a failure of this ongoing commitment." meets this requirement, unless we're saying that "verifiable" only means that the community will notice if/when something goes wrong that implies that the commitment was not met, and that seems pointless.

Reply:
I also find this section too perscriptive and would request to remove point 3.